main.go

```diff
@@ -16,14 +17,19 @@ func main() {
 	// run options
 	runTimeout := flag.Duration("run", 30*time.Second, "stop run after this duration")
 	flushTimeout := flag.Duration("flush", 10*time.Second, "wait timeout for agent flush")
+	seed := flag.Int64("seed", time.Now().Unix(), "random seed")
```

Comment:
why this needs to be a flag?
i think it would be great to minimize the number of flags so it doesn't become too hard to use